WORKSPACE

```diff
@@ -160,6 +163,7 @@ vizier_api_install_deps()
 
 pip_parse(
     name = "pxapi_python_doc_deps",
+    python_interpreter_target = interpreter,
```

Comment:
What's going on here?

Reply:
It ensures that the `pip_parse` uses the toolchain's pip rather than the system pip. Without this upgrading the dev container to 24.04 opts us into a newer pip and python version and causes a ton of build failures (lots of wheels fail to build for the versions pinned in the `requirements.bazel.txt` files).

We already did this for [vizier_api_python_deps](https://github.com/pixie-io/pixie/blob/0888c38df2dc414a36cc84198c7b159c39eea0e0/WORKSPACE#L153), it just wasn't maintained for the new `pip_parse` rules added since the 22.04 upgrade.